README.md

```diff
@@ -48,6 +54,25 @@ apt install polkadot
 
 ```
 
+#### Install using a user with elevated privileges (sudo)
+
+Run the following commands as a `non-root` user.
+
+```bash
+# Import the security@parity.io GPG key
+PARITY_SEC_KEY=9D4B2B6EB8F97156D19669A9FF0812D491B96798
+gpg --recv-keys --keyserver hkps://keys.mailvelope.com $PARITY_SEC_KEY
+sudo gpg --export $PARITY_SEC_KEY | sudo tee /usr/share/keyrings/parity.gpg
```

Comment:
```suggestion
gpg --export $PARITY_SEC_KEY | sudo tee /usr/share/keyrings/parity.gpg
```

Reply:
He wrote in the doc:
> You can either run the installation as the root user or run the
installation as a non-root user with elevated privileges

so it looks like all the examples are as root. It would be better however to consider an install as non root as @coderobe suggests.